ExampleMod/ExamplePlayer.cs

```diff
@@ -634,6 +635,103 @@ public class ExamplePlayer : ModPlayer
+			// By comparing that to drawPlayer.body, which represents the currently visible body armor, we make sure it's safe to draw ours
+			// Here we abort if the currently equipped body armor isn't what we need for our glowmask
+			if (drawPlayer.body != mod.GetEquipSlot("ExampleBreastplate", EquipType.Body)) {
+				return;
+			}
+
+			// Because we have a breastplate glowmask, and breastplates have different textures depending on if the player is male or female, we have to use two textures here
+			string gender = "";
+			if (!drawPlayer.Male) {
+				gender = "Female";
+			}
+
+			// The texture we want to display on our player
+			Texture2D texture = mod.GetTexture("Items/Armor/ExampleBreastplate_" + gender + "Body_Glowmask");
+
+			// The following variables (until drawData) are all copied 1:1 from vanilla code that handles breastplate drawing.
+			// It is advised to always copy the vanilla code that handles a particular EquipType, if you wish to make a glowmask for it,
+			// so it seamlessly overlaps
+			float drawX = (int)drawInfo.position.X + drawPlayer.width / 2;
+			float drawY = (int)drawInfo.position.Y + drawPlayer.height - drawPlayer.bodyFrame.Height / 2 + 4f;
+
+			// The rotation origin of the texture
+			// Because we are drawing a breastplate and it has the same alignment as the players' body, we use bodyOrigin
+			Vector2 origin = drawInfo.bodyOrigin;
+
+			// The position this texture will be drawn at on the screen
+			// Because we are drawing a breastplate and it has the same alignment as the players' body, we use bodyPosition
+			Vector2 position = new Vector2(drawX, drawY) + drawPlayer.bodyPosition - Main.screenPosition;
+
+			// The alpha of the texture when the player is blinking (immune)
+			// Should stay like that
+			float alpha = (255 - drawPlayer.immuneAlpha) / 255f;
+
+			// The color of the glowmask, pick what you want
+			Color color = Color.White;
+
+			// The frame that gets cut out from our texture that then will be drawn
+			// Because we are drawing a breastplate and it has the same alignment as the players' body, we use bodyFrame
+			Rectangle frame = drawPlayer.bodyFrame;
+
+			// The texture rotation around the origin
+			// Because we are drawing a breastplate and it's tied to players' body, we use bodyRotation
+			float rotation = drawPlayer.bodyRotation;
+
+			// The horizontal/vertical flipping (based on direction, gravity)
+			// Should stay like that
+			SpriteEffects spriteEffects = drawInfo.spriteEffects;
+
+			DrawData drawData = new DrawData(texture, position, frame, color * alpha, rotation, origin, 1f, spriteEffects, 0);
+
+			// The shader that's applied to the texture
+			// Depending on what EquipType this glowmask is representing, you need to assign the proper shader to it
+			// (This references the dye)
+			// Because this is a breastplate, we want to use the dye that is in the breastplate slot
+			drawData.shader = drawInfo.bodyArmorShader;
```

Comment:
Good to note you can also apply different shaders if you want, giving special glowmask effects. Sky is the limit.